fix(ci): subgraph#test が OOM kill (exit 137) で fail / turbo concurrency 制限で構造的解消

## 概要

PR #2994 (sdk test 修復) merge 後の master CI `Test (22.x)` job で `@niji/subgraph#test` (`graph test`) が **OOM kill (exit code 137)** で fail。 sdk test は全 27 pass、 turbo cache fix (PR #2992) で contracts test も実行されるようになった結果、 CI runner (GitHub Actions ubuntu-latest 7GB RAM) のメモリ消費が限界超え。

## 課題

- `@niji/subgraph:test` が SIGKILL (137) で停止
- `pnpm run test was killed (exit code 137), likely due to running out of memory` log で OOM 確定
- ローカル (Apple Silicon 32GB) では再現せず、 CI 環境特化問題

## 詳細

エラー抜粋 (run 28432148914 / job 84250305828)。

```
[ERROR] command finished with error: command (/home/runner/work/niji-monorepo/niji-monorepo/packages/niji-subgraph)
/home/runner/setup-pnpm/node_modules/.bin/pnpm run test was killed
(exit code 137), likely due to running out of memory
```

直前まで以下 task が並列実行されていた可能性。
- `@niji/contracts#test` (hardhat test) ... ethers + chai + mocha 構成、 ピーク 1-2GB
- `@niji/subgraph#test` (graph test) ... matchstick wasm runtime、 ピーク 1-3GB
- `@niji/sdk#test` (vitest run) ... happy-dom + Polly、 ピーク 500MB-1GB
- `@niji/webapp#test` (vitest run) ... happy-dom + 大量 component、 ピーク 1-2GB

合計で 4-8GB peak、 ubuntu-latest 7GB RAM では足りない。

## 解決方法

3 候補。

1. **turbo concurrency 制限** ... `pnpm turbo run test --concurrency=1` で逐次実行に変更 (CI のみ)
2. **runner upgrade** ... `runs-on: ubuntu-latest-4-cores` (16GB RAM) or `ubuntu-latest-8-cores` (32GB RAM) に変更、 ただし billing 増加
3. **NODE_OPTIONS** ... `--max-old-space-size=4096` を CI env に設定、 ただし wasm runtime には効かない可能性

推奨は #1 (concurrency 制限)。 build time +1-2 分の引き換えに OOM 構造的解消、 billing 増加なし。

## 仕組み

```mermaid
graph LR
    A[GitHub Actions runner 7GB RAM] --> B[turbo run test 並列]
    B --> C[contracts + subgraph + sdk + webapp 同時]
    C -->|peak 4-8GB| D[OOM kill 137]
    E[turbo run test --concurrency=1] --> F[逐次実行]
    F -->|peak 1-3GB| G[OOM 回避]
```

## 進め方

### Phase 1 — 調査・方針決定
- [ ] T001 `.github/workflows/node.yaml` の Test job 設定確認
- [ ] T002 turbo の concurrency option / NODE_OPTIONS 経路の優劣検討
- [ ] T003 採用案を 3 候補から決定 (推奨 = concurrency=1)

### Phase 2 — 実装
- [ ] T004 採用案を CI workflow に反映 `.github/workflows/node.yaml`

### Phase 3 — 検証・提出
- [ ] T005 PR push 後 CI `Test (22.x)` green を merge 前確認 (master 連鎖 fail から脱却)
- [ ] T006 PR 作成

## 完了条件

- [ ] master の `Test (22.x)` CI が green (全 11 task success)
- [ ] OOM kill 137 が再発しない
- [ ] build time 増加が許容範囲内 (+2 分以内目標)

## 影響範囲

- `.github/workflows/node.yaml` (Test job 設定変更 1-2 行)
- CI build time (concurrency=1 で +1-2 分見込み)

## 関連

- PR #2994 (#2993 / sdk image.test + Polly fixture、 本 Issue の前段)
- PR #2992 (#2989 / turbo cache outputs、 contracts test 動作回復で OOM 顕在化)
- PR #2987 (#2986 / hardhat-abi-exporter)
- PR #2985 (CAR-301 / turbo subgraph→contracts)
- 失敗 run = https://github.com/Niji-DAO/niji-monorepo/actions/runs/28432148914

--- a/package.json
+++ b/package.json
@@ -11,7 +11,7 @@
     "format": "NODE_OPTIONS=--max_old_space_size=16384 prettier --write 'packages/**/*.{ts(x)?,sol,md,css,json}' '!**/typechain/**'",
     "lint": "eslint --cache --cache-location .cache/.eslintcache",
     "prepare": "husky",
-    "test": "turbo run test"
+    "test": "turbo run test --concurrency=1"
   },
   "devDependencies": {
     "@eslint-react/eslint-plugin": "^2.13.0",
@@ -66,13 +66,13 @@
       "unix-dgram"
     ],
     "overrides": {
-      "@typescript-eslint/utils": ">=8.56.0",
       "@babel/traverse": ">=7.23.2",
       "@openzeppelin/contracts": "4.4.0",
       "@openzeppelin/contracts-upgradeable": "4.4.0",
       "@openzeppelin/hardhat-upgrades": ">=3.9.1",
       "@openzeppelin/upgrades-core": ">=1.42.0",
       "@types/node": "catalog:",
+      "@typescript-eslint/utils": ">=8.56.0",
       "async": ">=2.6.4",
       "axios": ">=0.30.0",
       "braces": ">=3.0.3",

